features/user_registration.feature

```diff
@@ -0,0 +1,26 @@
+Feature: User registration
+  As a user,
+  In order to have be able to finalise an order
+  I would like to be able to register an account
+
+  Background:
+    Given I visit the site
+    And I click the "Login" button
+
+  Scenario: Visitor can create an account to become a registered user[Hapy path]
+    And I click the "Sign up" button
+    Then I fill the "Name" with "John"
```

Comment:
Please refactor to
`And I fill the "Name" with "John"
`

Reply:
Corrected